docs/dotnet-configuration.md

```diff
@@ -4,6 +4,91 @@ title: Scope .NET Agent configuration
 sidebar_label: Configuration
 ---
 
+The behaviour of the Scope Agent can be modified by using  environment variables or by using a configuration file `Scope.yml`in the source root of the project.
+
+> These configuration settings can also be provided via CLI flags. Run `scope-run --help` for more information.
+
+
+
+## Disabling Scope instrumentation
+
+You can disable Scope auto instrumentation. Scope auto instruments your code and tests by default, you can disable this behaviour.
+
+### Using Environment Variables
+
+```sh
+SCOPE_AUTO_INSTRUMENT=False
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  auto_instrument: false
+  
+```
+
+
+
+## Changing service name
+
+You can change the name of the service when sending data to Scope. Scope initially uses `default` for the service name.
+
+### Using Environment Variables
+
+```sh
+SCOPE_SERVICE="service name"
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  service: 'service name'
+```
+
+
+
+## Setting Scope as Global Tracer
+
+You can set Scope as the Global Tracer of Open Tracing. Scope uses Open Tracing internally to handle Span and traces, using its own tracer, You can set it as the global tracer so it will capture all the spans generated by any Open Tracing library in relation to your tests. 
+
+### Using Environment Variables
+
+```sh
+SCOPE_SET_GLOBAL_TRACER=True
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  set_global_tracer: true
+  
+```
+
+
+
+## Change testing mode
+
+These variable indicates to the agent if it is running tests (`true`) or it is being used for runtime instrumentation (`false`).
+
+### Using Environment Variables
+
+```sh
+SCOPE_TESTING_MODE=False
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  testing_mode: false
+  
+```
+
+ By default `SCOPE_TESTING_MODE` property depends on whether the build has been triggered by a CI server (`true`), or not (`false`).
```

Comment:
`SCOPE_TESTING_MODE` property? Isn't that a env variable? 

What we want to say here is probably something along these lines: 
> If `SCOPE_TESTING_MODE` is not specified, the agent sets it to true when running in CI and false otherwise.

Reply:
The name changed to: `SCOPE_MODE_TESTING` to follow the `scope.yml`:
```yaml
scope:
    mode:
        testing: true
```

Reply:
changed